services/web-app/src/components/buttons/basicButton.tsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import { Button } from "@radix-ui/themes";
```

Comment:
[Note] I'm slightly worried that we are using both tailwind and radix- I think we should choose just one as otherwise I think we are going to run into problems in future (with them fighting each other etc)